qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -311,65 +311,56 @@ def observable(me_table, init_term=0, mapping="jordan_wigner"):
     return structure.convert_observable(jordan_wigner(mb_obs))
 
 
-def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_active_orbitals=None):
-    r"""Reads the Hartree-Fock (HF) electronic structure data file, defines an active
-    space and generates the table of matrix elements required to build the second-quantized
-    spin-projection operator :math:`\hat{S}_z`.
+def spin_z(n_orbitals, mapping="jordan_wigner"):
+    r"""Computes the total spin projection operator :math:`\hat{S}_z` in the Pauli basis.
 
     The second-quantized operator :math:`\hat{S}_z` reads,
 
     .. math::
 
         \hat{S}_z = \sum_{\alpha, \beta} \langle \alpha \vert \hat{s}_z \vert \beta \rangle
-        ~ \hat{c}_\alpha^\dagger\hat{c}_\beta,
+        ~ \hat{c}_\alpha^\dagger \hat{c}_\beta,
 
         \langle \alpha \vert \hat{s}_z \vert \beta \rangle = m_\alpha \delta_{\alpha,\beta},
```

Comment:
Maybe slightly nicer to place both equations on the same line?

![image](https://user-images.githubusercontent.com/2959003/87751974-69fdf500-c83e-11ea-8dbb-a256d9fa1545.png)